inc_internal/ziti_ctrl.h

```diff
@@ -63,6 +69,23 @@ void ziti_ctrl_get_well_known_certs(ziti_controller *ctrl, void (*cb)(char *, zi
 void ziti_ctrl_enroll(ziti_controller *ctrl, const char *method, const char *token, const char *csr,
                       void (*cb)(ziti_enrollment_resp *, ziti_error *, void *), void *ctx);
 
+void ziti_ctrl_pr_post_domain(ziti_controller *ctrl, char *id, char *domain,
+                              void (*cb)(void *, ziti_error *, void *),
```

Comment:
a typedef would be nice here for the callback type (especially when using from Swift, makes syntax much cleaner)

Reply:
tho we won't access these methods directly from Swift, so it wount make any difference :)